cadence/lib/js/src/kibble.js

```diff
@@ -0,0 +1,143 @@
+import { Address, UFix64 } from "@onflow/types";
+import {
+	deployContractByName,
+	executeScript,
+	getContractAddress,
+	getScriptCode,
+	getTransactionCode,
+	mintFlow,
+	sendTransaction,
+} from "flow-js-testing";
+
+import { getRegistry } from "./common";
+
+/*
+ * Deploys FungibleToken and Kibble contracts to Registry.
+ * @throws Will throw an error if transaction is reverted.
+ * @returns {Promise<*>}
+ * */
+export const deployKibble = async () => {
+	const Registry = await getRegistry();
+	await mintFlow(Registry, "10.0");
+
+	await deployContractByName({ to: Registry, name: "FungibleToken" });
+
+	const addressMap = { FungibleToken: Registry };
+	return deployContractByName({ to: Registry, name: "Kibble", addressMap });
```

Comment:
Not something to change for this PR, but I think it'd be good to explore using `flow.json` to lookup contracts by name, rather than implicit resolution to the `contracts` folder.